js/damage.js

```diff
@@ -86,6 +86,8 @@ function getDamageResult(attacker, defender, move, field) {
         description.moveType = move.type;
     } else if (move.name === "Judgment" && attacker.item.indexOf("Plate") !== -1) {
         move.type = getItemBoostType(attacker.item);
+    } else if (move.name === "Techno Blast" && attacker.item.indexOf("Drive") !== -1) {
+        move.type = getItemBoostType(attacker.item);
```

Comment:
This isn't really what I meant; you can keep your other function but just implement it like this function. Doing it in the same function will cause Drives to boost same-type attacks like Plates and the rest of them. [For reference](https://github.com/Zarel/honko-damagecalc/pull/30/files#diff-63ddc2b0f3b50a57a4f0b5e52d659438R281).